### PR TITLE
Change member table UI (fixes #6447)

### LINKED ISSUE #6447: DialogsAddTable: Member table should not include "Action"
### Current Behavior

There are three UI changes for this dialog:
1. The dialog for adding a member includes the "Action" column with one button per row to "View Profile".  The button fortunately does not perform an action, but it should be removed regardless.
2. The profile image has a pointer icon when the cursor hovers over which should not be there
3. A user can remove roles from another member from this view.

### Desired Behavior

1. The "Action" column should not be there in the dialog.
2. The cursor should remain default on the image.
3. No roles should be changed in this view.

Note that dialog is **DialogsAddTableComponent** with `mode` set to `'users'`.  That has **UsersComponent** as a child which in turn has **UsersTableComponent** as a child.

image

### PR DESCRIPTION
**Pull request by Haoran-Hu**
fixes #6447

1. The "Action" column should not be there in the dialog.
2. The cursor should remain default on the image.
3. No roles should be changed in this view.

### CODE REVIEW

**Comment by Haoran-Hu** [comment]
@paulbert **Health Provider** does not show up after adding a role to a member.

File: src/app/users/users-table.component.html [hunk 1]
```diff
@@ -88,45 +88,15 @@
         <mat-chip-list>
           <mat-chip *ngIf="element.doc.roles.length > 0 || element.doc.isUserAdmin" i18n>Learner</mat-chip>
           <ng-container *ngFor="let role of element.doc.roles; index as i">
-            <mat-chip *ngIf="role !== 'learner'" [removable]="!element.doc.isUserAdmin" (removed)="removeRole(element.doc,i)">
+            <mat-chip *ngIf="role !== 'learner'">
```
Comment (lmmrssa) [inline src/app/users/users-table.component.html:91]: just need to add condition for `removable` no need to change `removed`

File: src/app/users/users-table.component.html [hunk 2]
```diff
@@ -88,45 +88,15 @@
...
               <planet-role [role]="role"></planet-role>
-              <mat-icon matChipRemove *ngIf="isUserAdmin && !element.doc.isUserAdmin">cancel</mat-icon>
```
Comment (lmmrssa) [inline src/app/users/users-table.component.html:93]: just add condition on if

File: src/app/users/users-table.component.html [hunk 3]
```diff
@@ -88,45 +88,15 @@
...
             </mat-chip>
           </ng-container>
           <mat-chip *ngIf="element.doc.isUserAdmin && element.doc.roles.length === 0">Admin</mat-chip>
         </mat-chip-list>
         <span *ngIf="!element.doc.isUserAdmin && element.doc.roles.length === 0" i18n>Inactive</span>
       </mat-cell>
     </ng-container>
-    <ng-container matColumnDef="action">
```
Comment (lmmrssa) [inline src/app/users/users-table.component.html:101]: column should still be there but should be controlled by displayedcolumns property from component

File: src/app/users/users-table.component.ts [hunk 4]
```diff
@@ -41,7 +41,7 @@ export class TableState {
 })
 export class UsersTableComponent implements OnInit, OnDestroy, AfterViewInit, OnChanges {
 
-  @Input() displayedColumns = [ 'select', 'profile', 'name', 'visitCount', 'joinDate', 'lastLogin', 'roles', 'action' ];
+  @Input() displayedColumns = [ 'select', 'profile', 'name', 'visitCount', 'joinDate', 'lastLogin', 'roles' ];
```
Comment (lmmrssa) [inline src/app/users/users-table.component.ts:44]: should be based on condition

Comment (Haoran-Hu) [reply src/app/users/users-table.component.ts]: @lmmrssa 'action' is an element in `displayedColumns`, how to exclude 'action' based on condition. Is that `{action:isDialog}`

Comment (Haoran-Hu) [reply src/app/users/users-table.component.ts]: @lmmrssa I switch the status of `displayedColumns` in `user-table.component.ts`. I think it is working.

**Pull request merged**